src/DB/projects.json

```diff
@@ -265,5 +265,12 @@
     "title": "Movie Trailer",
     "description": "The movie-trailer app is a web application that allows users to search and view trailers for their favorite movies. The application is developed using React and integrated with the TMD and YouTube APIs to fetch movie trailers. The application has a responsive design that adapts to different screen sizes and features a search bar and filter option to help users navigate through the content.",
     "tech": ["React", "CSS", "JavaScript", "HTML", "TMD API", "Youtube API"]
+  },
+  {
+    "gh-username": "vishnu-mouli-102408",
+    "link": "spotify-clone-102408.netlify.app",
```

Comment:
add a valid link. it's not working.

Reply:
Bro it's working. I tried it now.plz tell me which error it's showing.so that I can resolve it.

Reply:
you can add github link

Reply:
I changed it bro.